src/errors.ts

```diff
@@ -0,0 +1,56 @@
+  PaymentFileInlucedWithPoolRegError: () => 'PaymentFileInlucedWithPoolRegError: Unexpected payment hardware signing file with pool registration certificate found',
+  MultipleStakingSigningFilesWithPoolRegError: () => 'MultipleStakingSigningFilesWithPoolRegError: Multiple staking signing files with pool registration certificate found, expected only one staking signing file',
+  MissingPaymentSigningFileError: () => 'MissingPaymentSigningFileError: Missing payment hardware signing file',
+  TooManySigningFilesError: () => 'TooManySigningFilesError: Too many signing files',
+  MissingStakingSigningFileError: () => 'MissingStakingSigningFileError',
+  MissingInputError: () => 'MissingInputError: Missing input',
+  MissingOutputError: () => 'MissingOutputError: Missing output',
+  TrezorError: () => 'TrezorError: Trezor operation failed, please make sure you are using the latest version of Trezor firmware',
+  TxInputParseError: () => 'TxInputParseError: Failed to parse input',
+  TxOutputParseError: () => 'TxOutputParseError: Failed to parse output',
+  WithrawalsParseError: () => 'WithrawalsParseError: Failed to parse withdrawals',
+  TxStakingKeyRegistrationCertParseError: () => 'TxStakingKeyRegistrationCertParseError: Failed to parse staking key registration certificate',
+  TxStakingKeyDeregistrationCertParseError: () => 'TxStakingKeyDeregistrationCertParseError: Failed to parse staking key deregistration certificate',
+  TxDelegationCertParseError: () => 'TxDelegationCertParseError: Failed to parse delegation certificate',
+  TxStakepoolRegistrationCertParseError: () => 'TxStakepoolRegistrationCertParseError: Failed to parse stakepool registration certificate',
+  TxSingleHostIPRelayParseError: () => 'TxSingleHostIPRelayParseError: Failed to parse single host IP relay',
+  TxSingleHostNameRelayParseError: () => 'TxSingleHostNameRelayParseError: Failed to parse single host name relay',
+  TxMultiHostNameRelayParseError: () => 'TxMultiHostNameRelayParseError: Failed to parse multi host name relay',
+  MissingSigningFileForCertficateError: () => 'MissingSigningFileForCertficateError: Missing signing file for certficate',
+  OwnerMultipleTimesInTxError: () => 'OwnerMultipleTimesInTxError: Owner multiple times in tx',
+  UnsupportedRelayTypeError: () => 'UnsupportedRelayTypeError: Unsupported relay type',
+  UnknownCertificateError: () => 'UnknownCertificateError: Unknown certificate',
+  UnsupportedCertificateTypeError: () => 'UnsupportedCertificateTypeError: Unsupported certificate type',
+  MissingSigningFileForWithdrawalError: () => 'MissingSigningFileForWithdrawalError: Missing signing file for withdrawal',
+}
+
+const getErrorTranslation = (
+  error: Error,
+): string => {
+  const translation = errors[error.name]
+  if (translation !== undefined) {
+    return translation(error)
+  }
+
+  return 'UknownError: An unkwown error has occured'
```

Comment:
The default message isn't very helpful.

Reply:
Removed, original error is now caught instead.